cluster/gke/util.sh

```diff
@@ -68,12 +69,30 @@ function test-build-release() {
 
 # Verify needed binaries exist.
 function verify-prereqs() {
-  echo "... in verify-prereqs()" >&2
-
-  ${GCLOUD} preview --help >/dev/null || {
-    echo "Either the GCLOUD environment variable is wrong, or the 'preview' component"
-    echo "is not installed. (Fix with 'gcloud components update preview')"
-  }
+  local cmd
```

Comment:
cmd is never set, and shouldn't be needed. replace occurrences of ${cmd} with gcloud.